tinygrad/codegen/llvmir.py

```diff
@@ -101,7 +101,7 @@ def uops_to_llvm_ir(uops:List[UOp], bufs:List[LazyBuffer]) -> str:
       element = lvars[vin[0]]
       if func_dtypes[0] != ir.FloatType():
         if dtypes.is_int(bufs[args.i].dtype):
```

Comment:
This was the reason I changed syntax from `is_int` to `is_signed_int`. This used to be `is_int`, which only checked for signed type, which I believe to be a bug. Since I changed `is_int` to encapsulate both signed and unsigned type, this should now be correct. No idea about the implications